test.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Vega-Embed test</title>
+  <style>
+    .vega-actions a {
+      padding: 5px;
+    }
+  </style>
+</head>
+<body>
+
+<div id="vis"></div>
+<script src="node_modules/d3/build/d3.min.js" charset="utf-8"></script>
```

Comment:
If we change to d3-selection and compile it into our lib, we don't need d3 here any more.